standalone_self_upgrade.cf.in

```diff
@@ -161,7 +161,7 @@ bundle agent cfengine_software_cached_locally
         string => "$(cfengine_software.package_dir)";
 
       # TODO Add shortcut for this path
-      "master_software_location"  string => "/var/cfengine/master_software_updates",
+      "master_software_location"  string => "$(sys.workdir)/master_software_updates",
```

Comment:
This one is OK, its from the perspective of the hub, used for downloading all software to the hub. But that little known feature might be broken now with other changes to packaging.